### PR TITLE
[PHP] improving php CS

### PR DESCRIPTION
**Pull request by JB-Ginguene**

### CODE REVIEW

**Review by utarwyn — COMMENTED** [review]
I think reviews I have made in https://github.com/green-code-initiative/ecoCode/pull/197 are also relevants in this PR :+1:

File: src/AvoidUsingGlobalVariablesCheck.php [hunk 1]
```diff
@@ -1,16 +1,21 @@
 <?php
+
 $a = 1;
 $b = 2;
-function somme() {
+function somme() { // NOK {{Prefer local variables to globals}}
```
Comment (dedece35) [inline src/AvoidUsingGlobalVariablesCheck.php:5]: the error message is at wrong place ... if you check in real environment with your local SonarQube, you will see that the error is placed at line 7 (like before your commit)

File: src/AvoidTryCatchFinallyCheck_NOK_FailsAllTryStatements.php [hunk 2]
```diff
@@ -7,25 +7,22 @@ function test() {
     throw new SpecificException('Oopsie');
 }
 
-try
-{
-  $picture = PDF_open_image_file($PDF, "jpeg", $imgFile, "", 0); // This is the original statement, this works on PHP4
+try { // NOK {{Avoid using try-catch}}
```
Comment (dedece35) [inline src/AvoidTryCatchFinallyCheck_NOK_FailsAllTryStatements.php:10]: the error message is at wrong place ... if you check in real environment with your local SonarQube, you will see that the error is placed at line 15 (like before your commit)

**Review by dedece35 — CHANGES_REQUESTED** [review]

File: src/AvoidGettingSizeCollectionInLoop.php [hunk 3]
```diff
@@ -115,7 +115,7 @@
```
Comment (dedece35) [inline src/AvoidGettingSizeCollectionInLoop.php:115]: in this file, all Non Compliant messages are at wrong position ... they should be positionned on the last "}" and not at the beginning of the loop.
could you please modify them, please ?

**Review by dedece35 — APPROVED** [review]

**Pull request merged**